src/pages/index.js

```diff
@@ -121,13 +250,23 @@ cardsList.renderItems();
 
 addCardButton.addEventListener("click", () => {
   formValidators["card-form"].resetValidation();
+  popupWithForms["add-card-modal"].resetButtonText();
   popupWithForms["add-card-modal"].open();
 });
 
 function createCard(inputValues) {
   const name = inputValues["image-title"];
   const link = inputValues["image-link"];
-  cardsList.addItem(generateCard({ name, link }));
+  //cardsList.addItem(generateCard({ name, link }));
+  api
+    .addCard(inputValues["image-title"], inputValues["image-link"])
+    .then((res) => {
+      cardsList.addItem(generateCard(res));
+      popupWithForms["add-card-modal"].changeButtonText();
+    })
+    .catch((err) => {
+      console.error(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors